.agents/evals/check-deliverables.sh

```
@@ -0,0 +1,240 @@
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+
+if [[ -d "${FRONTEND_DIR}" ]]; then
+  ok "Module directory exists: ${FRONTEND_DIR}"
+  
+  # Check for common components
+  COMPONENTS=$(find "${FRONTEND_DIR}" -name "*.tsx" -not -name "*.test.tsx" | sort)
+  if [[ -n "${COMPONENTS}" ]]; then
+    echo ""
+    echo "Components found:"
+    echo "${COMPONENTS}" | while read -r comp; do
+      echo "  • $(basename ${comp})"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)